content/blog/2018/custom-gateway/index.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+publishdate: 2018-10-26
+subtitle: Custom Ingress Gateway
+attribution: Julien SENON
+weight: 89
+keywords: [ingress,traffic-management, aws]
+---
+
+This post provide instructions to create manually a new custom ingress gateway with automatic provisioning of certificate based on cert-manager.
+
+This post was validated on AWS Infrastructure
+
+## Prerequisites
+
+Have cert-manager installed with [chart](https://github.com/helm/charts/tree/master/stable/cert-manager)
+
+## Cert-Manager Configuration
+
+### Check installation
```

Comment:
Two headings without content in between indicate a problem with the content structure. I see what the content structure problem here actually is. In this case, all level 3 headings should be items in a numbered list. They are the steps you are taking to configure cert-manager.
Thus, the heading becomes part of the text on my comment in line 23.